drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -158,6 +162,50 @@ static esp_err_t eth_esp32_rx(void *buffer, uint16_t len, void *eb)
 	return -EIO;
 }
 
+#ifdef CONFIG_ESP32_WIFI_AP_STA_MODE
+static esp_err_t eth_esp32_ap_mode_rx(void *buffer, uint16_t len, void *eb)
```

Comment:
Shall we remove the `eth_` prefix from both callback functions?
`eth_esp32_ap_mode_rx`-> `wifi_esp32_ap_mode_rx`
`eth_esp32_rx` -> `wifi_esp32_sta_rx`

Reply:
Yes, I think we should to reflect these `rx` function are for `wi-fi` not for `eth`